pkg/agent/endpoints/sds/handler.go

```diff
@@ -364,10 +376,16 @@ func buildTLSCertificate(identity cache.Identity) (*any.Any, error) {
 	})
 }
 
-func buildValidationContext(bundle *bundleutil.Bundle) (*any.Any, error) {
+func buildValidationContext(bundle *bundleutil.Bundle, defaultBundleNameBool bool, defaultBundleName string) (*any.Any, error) {
+	var id string
+	if defaultBundleNameBool {
+		id = defaultBundleName
+	} else {
+		id = bundle.TrustDomainID()
+	}
```

Comment:
nit: "name" is probably a more appropriate variable name. Also, how would you feel about removing the else branch?
```suggestion
        name := identity.Entry.SpiffeId
	if defaultBundleNameBool {
		name = defaultBundleName
	}
```